public_html/wp-content/themes/wporg-events-2023/patterns/front-cover.php

```diff
@@ -46,7 +46,7 @@
 <!-- wp:column {"width":"33.33%","fontSize":"medium","fontFamily":"eb-garamond"} -->
 <div class="wp-block-column has-eb-garamond-font-family has-medium-font-size" style="flex-basis:33.33%"><!-- wp:group {"style":{"border":{"radius":"2px","width":"1px"},"spacing":{"padding":{"left":"var:preset|spacing|30","top":"var:preset|spacing|20","bottom":"var:preset|spacing|20"}}},"borderColor":"light-grey-1","layout":{"type":"constrained"}} -->
 <div class="wp-block-group has-border-color has-light-grey-1-border-color" style="border-width:1px;border-radius:2px;padding-top:var(--wp--preset--spacing--20);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--30)"><!-- wp:paragraph {"style":{"layout":{"selfStretch":"fit","flexSize":null},"elements":{"link":{"color":{"text":"var:preset|color|charcoal-0"}}}},"textColor":"charcoal-0","fontSize":"heading-6"} -->
-<p class="has-charcoal-0-color has-text-color has-link-color has-heading-6-font-size">540,537 members</p>
+<p class="has-charcoal-0-color has-text-color has-link-color has-heading-6-font-size">540,537 attendees</p>
```

Comment:
I don't think it'd be accurate to change this. That number is the # of meetup.com group members (plus WordCamp attendees, but that doesn't affect it significantly), and the number of attendees is likely to be significantly different.

If we really want to say attendees, we could try to estimate the # based on RSVPs, but I don't think it'd be accurate enough to publish. IMO members is the most accurate/meaningful stat we can use here.

Reply:
Thanks for that context. I was going off the designs. I'll provide that feedback and switch it back to members today.

Reply:
So summing up the problem, if we use "attendees", we will be underrepresenting the actual number of attendees.

I also find `members` to be an inclusive term. What about just `people`? 

Simple is as simple does.

Reply:
🤔 that's a good point, we definitely don't want it to feel exclusionary. I like `participants`, thanks for thinking of that!  

I think the problem with `attendees` is more that it _over_represents the number of attendees, since so many people join a meetup group, and even RSVP to an event, but don't actually show up. IMO accuracy is the most important factor, and it'd be better to err on the side of too low than too high. My concern is that inflating stats, even unintentionally, would erode trust.

I'll go ahead and update it to `participants` since we're coming up on the deadline to launch, but I'm happy to continue discussing it if y'all would like to iterate on that, or make any changes.